string name
MementarTimestamp start_stamp
MementarTimestamp end_stamp
